README.md

```diff
@@ -35,6 +35,9 @@ To determine whether to allow response _response_ to a request _request_, run th
    1. Append (_request_'s opaque media identifier, _request_'s current URL) to the user agent's opaque-safelisted requesters set.
    1. Return true.
 1. If _nosniff_ is true, then return false.
+1. If _response_'s status is not an [ok status](https://fetch.spec.whatwg.org/#ok-status), then return false.
+1. If _mimeType_'s essence starts with "`audio/`", "`image/`", or "`video/`", then return false.
+1. If _mimeType_'s essence is "`text/csv`", then return false.
```

Comment:
This looks good, but maybe it would help if I would understand the motivation a bit more.  For example, ChromeOS handles both csv and PDF via a similar mechanism, so maybe we should look at both here (and other similar, [never-sniff MIME types](https://source.chromium.org/chromium/chromium/src/+/master:services/network/public/cpp/cross_origin_read_blocking.cc;l=251;drc=a122d0fda4ecbcf6fd184a5393be1d13b867a868) in Chromium's implementation of CORB)

Reply:
As per #6 these are existing things we block for classic scripts, including for same-origin requests. I think it's definitely reasonable to block even more types outright though if Chrome has found that possible. Let's discuss that in a new issue?

Reply:
I guess that this is just a performance optimization, so I am not sure how much effort is reasonable here.  I don't have a strong opinion either way (i.e. I am fine with proceeding with just the `text/csv` and other MIME types from https://fetch.spec.whatwg.org/#should-response-to-request-be-blocked-due-to-mime-type?)

Reply:
If we can block more MIME types outright I think that's definitely preferable, not just for performance. As it also means that resources with those MIME types can contain any arbitrary byte sequence, even those that happen to parse as JavaScript without error.